docs/topics/request-response.rst

```diff
@@ -508,6 +508,38 @@ method for this job. Here's an example spider which uses it::
 
             # continue scraping with authenticated session...
 
+JSONRequest
+-----------
+
+The JSONRequest class extends the base :class:`Request` class with functionality for
+dealing with JSON requests.
+
+.. class:: JSONRequest(url, [data, ...])
```

Comment:
Hey! I wonder how can we make argument docs more clear here - currently users may assume `data` is a second argument, while a second argument is callback.

Reply:
👍 
> is callback.

@kmike you probably meant "a keyword argument" here.

Reply:
I meant that one should be able to write `JsonRequest(url, self.parse_result, data=...)`

Reply:
Ah right this makes sense, sorry for the noise.

Reply:
Thanks @kmike for the review.

This is slightly changed now.